libs/mrwebrtc/src/peer_connection.cpp

```diff
@@ -186,9 +186,6 @@ class SessionDescObserver : public webrtc::SetSessionDescriptionObserver {
     RTC_LOG(LS_ERROR) << "Error setting session description: "
                       << error.message();
   }
-  void OnFailure(const std::string& error) override {
```

Comment:
Was this removed from the interface? This was quite helpful to diagnose issues.

Reply:
Yes, removed from the interface. the overload above should be good enough right?

Reply:
Ah sorry didn't see it. Yes fine.

Reply:
Do you know the right place to post this question? Hey:

Does anyone know how to upgrade my platform of  Google Glasses WEBRTC platform running on android 8.1 and AWS Kinesis?